src/core_plugins/kibana/public/discover/embeddable/search_embeddable.js

```diff
@@ -58,11 +66,24 @@ export class SearchEmbeddable extends Embeddable {
   initializeSearchScope() {
     this.searchScope = this.$rootScope.$new();
 
-    this.pushContainerStateParamsToScope();
-
     this.searchScope.description = this.savedSearch.description;
     this.searchScope.searchSource = this.savedSearch.searchSource;
 
+    const timeRangeSearchSource = this.searchScope.searchSource.clone();
+    timeRangeSearchSource.inherits(false);
+    timeRangeSearchSource.filter(() => {
+      return getTime(timeRangeSearchSource.index(), this.timeRange);
+    });
+
+    const filtersSearchSource = this.searchScope.searchSource.clone();
+    filtersSearchSource.inherits(timeRangeSearchSource);
```

Comment:
This one should still get `this.filters` and `this.query` in it?

Reply:
yes, this one will receive filters and query